### PR TITLE
Bench update

### PR DESCRIPTION
**Pull request by mloubout**
Second pass.

Homegenized names accross seismic examples.
Added (forgottten last pR) acoustic_ssa mode

### CODE REVIEW

File: examples/seismic/skew_self_adjoint/example_iso.py [hunk 1]
```diff
@@ -55,9 +55,9 @@ def run(shape=(50, 50, 50), spacing=(10.0, 10.0, 10.0), tn=1000.0,
     info("Applying Adjoint")
     solver.adjoint(rec, autotune=autotune)
     info("Applying Born")
-    solver.born(dm, autotune=autotune)
+    solver.jacobain(dm, autotune=autotune)
```
Comment (georgebisbas) [inline examples/seismic/skew_self_adjoint/example_iso.py:58]: jacobIAn typos?

Comment (georgebisbas) [reply examples/seismic/skew_self_adjoint/example_iso.py:58]: Are thos examples being tested?

Comment (FabioLuporini) [reply examples/seismic/skew_self_adjoint/example_iso.py:58]: indeed, good point. I'd say let's add a tiny test to test_benchmark.py

Comment (mloubout) [reply examples/seismic/skew_self_adjoint/example_iso.py:58]: Good catch. 
>. I'd say let's add a tiny test to test_benchmark.py
Already there with correct name, this is in the example

Comment (mloubout) [reply examples/seismic/skew_self_adjoint/example_iso.py]: fixed

**Pull request merged**